src/main/java/io/phdata/streamliner/schemadefiner/util/StreamlinerUtil.java

```diff
@@ -117,7 +125,7 @@ public static void writeConfigToYaml(Configuration outputConfig, String outputDi
 
     public static void writeConfigToYaml(ConfigurationDiff outputConfig, String outputDir) throws IOException {
         if (outputDir == null || outputDir.equals("")) {
-            outputDir = outputConfig.getName() + "/" + outputConfig.getEnvironment() + "/confDiff";
+            outputDir = "output/" + outputConfig.getName() + "/" + outputConfig.getEnvironment() + "/confDiff";
```

Comment:
String.format would clean this up nicely

Reply:
updated